app/src/main/kotlin/nl/q42/template/logging/CrashlyticsAntilog.kt

```diff
@@ -0,0 +1,40 @@
+package nl.q42.template.logging
+
+import com.google.firebase.crashlytics.ktx.crashlytics
+import com.google.firebase.ktx.Firebase
+import io.github.aakira.napier.Antilog
+import io.github.aakira.napier.DebugAntilog
+import io.github.aakira.napier.LogLevel
+
+/** A value suitable for Crashlytics */
+private const val MAX_CHARS_IN_LOG = 1200
+
+class CrashlyticsAntilog : Antilog() {
```

Comment:
could you maybe explain the name "antilog", it is not really clear to me (is probably me).

Reply:
It's not you, it's Napier. They don't explain it either.

Reply:
But.. there must be an explanation somewhere?

Reply:
"Napier" apparently was a Scottish mathematician who invented logarithms. 
The function to remove loggers is called "takeLogarithm"

Seems like this person has a weird sense of humor and maybe is a bit careless

https://github.com/AAkira/Napier/issues/18#issuecomment-515739881

Dunno man, I renamed it to CrashlyticsLogger

Reply:
i think he's just japanese lol (the r and l sounds are [very close](https://www.youtube.com/watch?v=2yzMUs3badc) in japanese so it's understandable the typo in https://github.com/AAkira/Napier/issues/18#issuecomment-515739544)

looking into how he's expressing himself in the readme it seems like a classic not very fluent japanglish way of writing things

about antilog, it's probably what nino says and he's full in with the "mathematician-theme" seeing he's trying to portray himself as _quirky_ and _fun_
```
I recommend using it if it supports kotlin multiplatform project.😜
```